app/tests/test_collection_model.py

```diff
@@ -51,20 +55,105 @@ def test_create_collection_invalid_providers(self):
         with self.assertRaises(
             ValidationError, msg="Collection with invalid providers was accepted."
         ):
-            CollectionFactory().create_sample(sample="collection-invalid-providers", db_create=True)
+            Factory().create_collection_sample(
+                sample="collection-invalid-providers", db_create=True
+            )
 
     def test_create_collection_with_providers_and_links(self):
         # try to create a valid collection with providers and links. Should not raise any errors.
-        CollectionFactory().create_sample(
+        Factory().create_collection_sample(
             name="collection-links-providers", sample="collection-1", db_create=True
         )
 
     def test_create_collection_only_required_attributes(self):
         # try to create a valid collection with only the required attributes.
         # Should not raise any errors.
-        CollectionFactory().create_sample(
+        Factory().create_collection_sample(
             name="collection-required-only",
             sample="collection-1",
             db_create=True,
             required_only=True
         )
+
+    def test_collection_update_on_item_write_operations(self):
+        # assert that collection's updated property is updated when an item is
+        # added to the collection, this item is updated and this item is deleted
+
+        # check collection's update on item insertion
+        initial_last_modified = self.collection.model.updated
+        sleep(0.01)
+        item = Factory().create_item_sample(self.collection.model, sample='item-1', db_create=True)
+        item.model.full_clean()
+        item.model.save()
+        self.collection.model.refresh_from_db()
+        self.assertGreater(
+            self.collection.model.updated,
+            initial_last_modified,
+            msg="Collection's updated property was not updated on item insert"
+        )
+
+        # check collection's update on item update
+        initial_last_modified = self.collection.model.updated
+        sleep(0.01)
+        item.model.name = "new_randon_name"
```

Comment:
Renaming item is not allowed ! Currently this check is only done at view level but if we change the validation in a lower level then this will failed. Update another attribute that is allowed instead of the name, e.g. the `properties_title`